src/breadcrumb/breadcrumb.component.ts

```diff
@@ -14,6 +14,7 @@ import { BreadcrumbItem } from "./breadcrumb-item.interface";
 import { BreadcrumbItemComponent } from "./breadcrumb-item.component";
 import { DomSanitizer } from "@angular/platform-browser";
```

Comment:
```suggestion
```

Could you remove DomSanitizer import & dependency injection in constructor as well, please. 

It isn't being used anywhere in the code & seems to have remained from previous changes. Thanks!